tailwind.config.js

```diff
@@ -20,10 +21,18 @@ module.exports = {
       "color-danger-hover": theme("colors.danger-hover"),
       "color-danger-press": theme("colors.danger-press"),
     }),
+    borderWidth: {
```

Comment:
This is going to be so useful!

Reply:
@julio8a border width is already part of the tailwind default. This is just adding 3px, just to clarify.

Reply:
I thought we agreed to kill the 3px? @bstifle ?

Reply:
We should be able to kill this in favor of extending.